<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="get">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>241</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотреть информацию о блюде</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dish_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>155</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dish4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dish_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dish3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dish_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dish2</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>9</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Меню на понедельник</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="dish_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dish5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="dish">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dish1</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Оценить </string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Информация от опрошенных </string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mon"/>
   <addaction name="tue"/>
   <addaction name="wed"/>
   <addaction name="thu"/>
   <addaction name="fri"/>
   <addaction name="sat"/>
   <addaction name="buffet"/>
  </widget>
  <action name="mon">
   <property name="text">
    <string>Пн</string>
   </property>
  </action>
  <action name="tue">
   <property name="text">
    <string>Вт</string>
   </property>
  </action>
  <action name="wed">
   <property name="text">
    <string>Ср</string>
   </property>
  </action>
  <action name="thu">
   <property name="text">
    <string>Чт</string>
   </property>
  </action>
  <action name="fri">
   <property name="text">
    <string>Пт</string>
   </property>
  </action>
  <action name="sat">
   <property name="text">
    <string>Сб</string>
   </property>
  </action>
  <action name="buffet">
   <property name="text">
    <string>Буфет</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
